boards/DIM/Inc/App/App_DimWorld.h

```diff
@@ -49,6 +49,32 @@ struct DimWorld *App_DimWorld_Create(
  */
 void App_DimWorld_Destroy(struct DimWorld *world);
 
+/**
+ * Get the current second for the given world
+ * @note The second is rounded down:
+ *          - If the current time is 999ms, this function returns 0 second.
+ *          - If the current time is 1001ms, this function returns 1 second.
+ * @param world The world to get current second for
+ * @return The current second for the given world
+ */
+uint32_t App_DimWorld_GetCurrentSecond(struct DimWorld *world);
```

Comment:
To facilitate cycling through errors at 1Hz like we discussed, we need to have the concept of time. I thought it would be appropriate to add time to `World` with these APIs.

Note that `App_DimWorld_SetCurrentMillisecond` is called in `Task1kHz` for ARM target but it's called manually in the tests.

Reply:
Whatever solution we end up going with, we should eventually implement it on other boards that also require time.

Reply:
Actually it was kinda late and I meant to write a separate shared time module and pass it into world. 

I.e.

App_DimWorld_GetTime()
App_SharedTime_SetCurrentMs()
App_SharedTime_GetCurrentMs()
App_SharedTime_GetCurrentSec()

Reply:
Yeah, this seems like an entirely reasonable solution to me.